status/index.go

```diff
@@ -0,0 +1,109 @@
+package status
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sync"
+
+	"github.com/egreb/websitestatus/site"
+)
+
+var state = make(map[string]CurrentStatus)
+var mutex = &sync.Mutex{}
+
+// Check the status for every site, if status is bad send notification to slack channel
+func Check(s []site.Status) {
+	for _, x := range s {
```

Comment:
For løkke i en så stor funksjon mener jeg er et tegn på at funksjonaliteten kan dras ut i en egen funksjon. 

```
func Check(s []site.Status) {
	for _, x := range s {
		check(x)
	}
}

func check(s site.Status) {
	...
}
```